subworkflows/nf-core/fasta_newick_epang_gappa/meta.yml

```diff
@@ -0,0 +1,82 @@
+name: "fasta_newick_epang_gappa"
+description: Run phylogenetic placement with a number of query sequences plus a reference alignment and phylogeny. Used in nf-core/phyloplace.
+keywords:
+  - phylogenetic placement
+  - phylogenetics
+  - alignment
+  - fasta
+  - newick
+modules:
+  - hmmer/hmmbuild
+  - hmmer/hmmalign
+  - hmmer/eslalimask
+  - hmmer/eslreformat
+  - epang/place
+  - epang/split
+  - gappa/examinegraft
+  - gappa/examineassign
+  - gappa/examineheattree
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - queryseqfile:
```

Comment:
this and all following are also in a map included, right? If so, I am not sure whether that explanation here is sufficient, probably one additional entry above queryseqfile could explain the general pattern, as you have in lines 19 of main.nf, followed by the detailed explanation that you have here.
In https://nf-co.re/docs/contributing/subworkflows#documentation point 2 is

> Structure: [ val(meta), path(tsv)]

as an example, maybe that yould be used here

Reply:
I agree here it's way more clear if the input channel structure is shown in the meta

Reply:
I completely agree, just didn't know how, so thanks for the pointer.

Reply:
I like it, I think its much clearer now. I added suggestions.